sections/clause_07a_sensing_core_entities.adoc

```diff
@@ -893,7 +889,7 @@ Each Feature entity SHALL have the mandatory properties and MAY have the optiona
 [%metadata]
 identifier:: {identifier}/req/datamodel/sensing/feature/relations
 
-Each Feature entity SHALL have the direct relation between a Feature entity and other entity types listed in Table {counter:table-num}.
+Each Thing entity SHALL have the direct relation between a Thing entity and other entity types listed in Table XX.
```

Comment:
I'm pretty sure this section is about Feature, not Thing?

Why change the `{counter:table-num}` to XX?
`{counter:table-num}` should give the number of the previous table.

Reply:
looks like a merge issue